src/discovery/Discovery.ts

```diff
@@ -17,22 +16,28 @@ import type Sigchain from '../sigchain/Sigchain';
 import type KeyManager from '../keys/KeyManager';
 import type { ClaimIdEncoded, Claim, ClaimLinkIdentity } from '../claims/types';
 import type { ChainData } from '../sigchain/types';
+import type TaskManager from '../tasks/TaskManager';
+import type { ContextTimed } from '../contexts/types';
+import type { PromiseCancellable } from '@matrixai/async-cancellable';
```

Comment:
Move this to the top. Imports from outside packages are above local imports.

Reply:
Moved.